lms/static/sass/base/_animations.scss

```diff
@@ -279,3 +279,138 @@
     opacity: 0;
   }
 }
+
+
+// +Fade In - Extend
```

Comment:
Not being included elsewhere, but there were duplicate "fade-ins" so I removed the oddball.